### ISSUE TITLE
Offset BMP280 sensor

### ISSUE THREAD

**Issue by JPBreestraat** [arendst/Sonoff-Tasmota#1425]
I installed a bmp280 next to a dht11. The output of the BMP280 sensor is a bit offset.
![schermafbeelding 2017-12-23 om 13 01 16](https://user-images.githubusercontent.com/34595274/34319459-9768370a-e7e3-11e7-82f7-3ca57ca5bf3f.png)

is there a way to correct it? ( for the dutch in jip en janneke taal)

thank you

**Comment by Frogmore42** [comment]
What elevation are you at?
With a pressure reading of 565 hPa, you are either at a very high elevation or have something very wrong.  Similarly, a temperature of -142.2 is probably not just a little offset, it is very wrong.  That either says things are hooked up incorrectly, you have a bad part, or something is wrong with the code.  I don't have a BMP280 so can't say which of those it might be.

**Comment by JPBreestraat** [comment]
the elevation is +3m, i also have a bmp180 and it's also offset. not that high as the bmp280 but it stil is(70C)

**Comment by Frogmore42** [comment]
It sounds like there is something significantly wrong with your setup, or your BMP280.  I discovered that I had a BMP280 in my parts bin.  So, I hooked it up and it is reading numbers that appear to be correct. 

![image](https://user-images.githubusercontent.com/14101627/34323741-91e14db6-e80c-11e7-81f5-35f116aedce4.png)

So, it looks like it is possible to get this working.  When I connected it to a NodeMCU device, it did not work at all, but that device has had issues in the past (not sure if it is the device or the breadboard that it is on).  I did not have to add any pullups on the I2C lines.
